klogr/klogr.go

```diff
@@ -68,24 +67,16 @@ type klogger struct {
 	format    Format
 }
 
-func (l klogger) clone() klogger {
-	return klogger{
-		level:  l.level,
-		prefix: l.prefix,
-		values: copySlice(l.values),
-		format: l.format,
-	}
-}
-
-func copySlice(in []interface{}) []interface{} {
-	out := make([]interface{}, len(in))
-	copy(out, in)
-	return out
+func (l *klogger) Init(info logr.RuntimeInfo) {
+	l.callDepth += info.CallDepth
 }
 
 // Magic string for intermediate frames that we should ignore.
 const autogeneratedFrameName = "<autogenerated>"
 
+// Cached depth of this interface's log functions.
+var framesAtomic int32 // atomic
```

Comment:
If you look at https://github.com/go-logr/glogr/commit/7a85ef23c9f6a08ad8444996c1d7250107b5e76c you will see that this logic id no longer needed since there isn't an interface in between.

Reply:
I meant that I think you don't need `autogeneratedFrameName` or `framesToCaller` or any of that :)

Reply:
whooooops! done.